velox/functions/prestosql/tests/DecimalArithmeticTest.cpp

```diff
@@ -370,6 +370,15 @@ TEST_F(DecimalArithmeticTest, decimalDivTest) {
               std::vector<int128_t>{DecimalUtil::kLongDecimalMax},
               DECIMAL(38, 0))}),
       "Decimal overflow: 99999999999999999999999999999999999999 * 10000");
+
+  // Rescale factor > max precision (38).
+  VELOX_ASSERT_THROW(
+      testDecimalExpr<TypeKind::HUGEINT>(
```

Comment:
We don't need to call testDecimalExpr here, just evaluate is sufficient (similar to the repro in the GitHub issue).

Reply:
Done.